src/games/supported/Galaxian.cpp

```diff
@@ -122,3 +122,33 @@ void GalaxianSettings::loadState(Deserializer & ser) {
   m_lives = ser.getInt();
 }
 
+// returns a list of mode that the game can be played in
+ModeVect GalaxianSettings::getAvailableModes() {
+    ModeVect modes = { 1, 2, 3, 4, 5, 6, 7, 8, 9 };
+    return modes;
+}
+
+// set the mode of the game
+// the given mode must be one returned by the previous function
+void GalaxianSettings::setMode(game_mode_t mode, System &system,
+                              std::unique_ptr<StellaEnvironmentWrapper> environment) {
+
+    if (mode >= 1 && mode <= 9) {
+        // press select until the correct mode is reached
+        while (mode != static_cast<unsigned>(readRam(&system, 0xB3))) {
```

Comment:
In all the other games we are not using `static_cast<unsigned>()` when reading the RAM to compare with the mode value. Why are you doing it here?

Reply:
This is just to silence the (very reasonable) warning produced by gcc otherwise regarding comparing signed and unsigned integers. The cast is an explicit acknowledgement that that is what is happening. I'm happy to remove it if it's a problem. Let me know.